blinky_skeleton.py

```diff
@@ -1,10 +1,22 @@
 from edg import *
+from electronics_model.KiCadSchematicBlock import KiCadSchematicBlock
 
 
+class KiCadBlock(KiCadSchematicBlock):
```

Comment:
Can you back out the changes to blinky_skeleton? This is meant as an example starter and should be clean.

It's fine to use it for quick and dirty testing (though I think using the unit test system is better), but quick and dirty code should not be merged back in as final code.

Reply:
Oops sorry forgot that I left that in. Just changed it back to what's in the master branch.